examples/medplum-patient-intake-demo/src/bots/core/intake-utils.ts

```diff
@@ -314,16 +314,34 @@ export async function addAllergy(
     return;
   }
 
-  const clinicalStatus = answers['allergy-clinical-status']?.valueCoding;
   const reaction = answers['allergy-reaction']?.valueString;
   const onsetDateTime = answers['allergy-onset']?.valueDateTime;
 
   await medplum.upsertResource(
     {
       resourceType: 'AllergyIntolerance',
+      clinicalStatus: {
+        text: 'Active',
+        coding: [
+          {
+            system: 'http://hl7.org/fhir/ValueSet/allergyintolerance-clinical',
+            code: 'active',
+            display: 'Active',
+          },
+        ],
+      },
+      verificationStatus: {
+        text: 'Unconfirmed',
+        coding: [
+          {
+            system: 'http://hl7.org/fhir/ValueSet/allergyintolerance-verification',
+            code: 'unconfirmed',
+            display: 'Unconfirmed',
+          },
+        ],
+      },
```

Comment:
@rahul1 This addresses the allergy statuses 
- https://github.com/medplum/medplum/pull/5033#discussion_r1713645075
- https://github.com/medplum/medplum/pull/5033#discussion_r1713645680